src/Kovan-DssSpell.sol

```diff
@@ -29,17 +37,131 @@ contract SpellAction {
+        require(FlipAbstract(MCD_FLIP_RENBTC_A).cat() == MCD_CAT, "flip-cat-not-match");
+        require(FlipAbstract(MCD_FLIP_RENBTC_A).ilk() == RENBTC_ILK, "flip-ilk-not-match");
+
+        SpotAbstract(MCD_SPOT).file(RENBTC_ILK, "pip", PIP_RENBTC);
+
+        // Set the RENBTC-A Flipper in the Cat
+        CatAbstract(MCD_CAT).file(RENBTC_ILK, "flip", MCD_FLIP_RENBTC_A);
+
+        // Init RENBTC-A ilk in Vat & Jug
+        VatAbstract(MCD_VAT).init(RENBTC_ILK);
+        JugAbstract(MCD_JUG).init(RENBTC_ILK);
+
+        // Allow RENBTC-A Join to modify Vat registry
+        VatAbstract(MCD_VAT).rely(MCD_JOIN_RENBTC_A);
+        // Allow the RENBTC-A Flipper to reduce the Cat litterbox on deal()
+        CatAbstract(MCD_CAT).rely(MCD_FLIP_RENBTC_A);
+        // Allow Cat to kick auctions in RENBTC-A Flipper
+        FlipAbstract(MCD_FLIP_RENBTC_A).rely(MCD_CAT);
+        // Allow End to yank auctions in RENBTC-A Flipper
+        FlipAbstract(MCD_FLIP_RENBTC_A).rely(MCD_END);
+        // Allow FlipperMom to access to the RENBTC-A Flipper
+        FlipAbstract(MCD_FLIP_RENBTC_A).rely(FLIPPER_MOM);
+        // Disallow Cat to kick auctions in RENBTC-A Flipper
+        // !!!!!!!! Only for certain collaterals that do not trigger liquidations like USDC-A)
+        // FlipperMomAbstract(FLIPPER_MOM).deny(MCD_FLIP_RENBTC_A);
+
+        // Allow OsmMom to access to the RENBTC Osm
+        // !!!!!!!! Only if PIP_RENBTC = Osm and hasn't been already relied due a previous deployed ilk
+        // OsmAbstract(PIP_RENBTC).rely(OSM_MOM);
+        // Whitelist Osm to read the Median data (only necessary if it is the first time the token is being added to an ilk)
+        // !!!!!!!! Only if PIP_RENBTC = Osm, its src is a Median and hasn't been already whitelisted due a previous deployed ilk
+        // MedianAbstract(OsmAbstract(PIP_RENBTC).src()).kiss(PIP_RENBTC);
+        // Whitelist Spotter to read the Osm data (only necessary if it is the first time the token is being added to an ilk)
+        // !!!!!!!! Only if PIP_RENBTC = Osm or PIP_RENBTC = Median and hasn't been already whitelisted due a previous deployed ilk
+        // OsmAbstract(PIP_RENBTC).kiss(MCD_SPOT);
+        // Whitelist End to read the Osm data (only necessary if it is the first time the token is being added to an ilk)
+        // !!!!!!!! Only if PIP_RENBTC = Osm or PIP_RENBTC = Median and hasn't been already whitelisted due a previous deployed ilk
+        // OsmAbstract(PIP_RENBTC).kiss(MCD_END);
+        // Set RENBTC Osm in the OsmMom for new ilk
+        // !!!!!!!! Only if PIP_RENBTC = Osm
+        OsmMomAbstract(OSM_MOM).setOsm(RENBTC_ILK, PIP_RENBTC);
+
+        // Set the global debt ceiling
+        VatAbstract(MCD_VAT).file("Line", 1234 * MILLION * RAD);
+        // Set the RENBTC-A debt ceiling
+        VatAbstract(MCD_VAT).file(RENBTC_ILK, "line", 2 * MILLION * RAD);
+        // Set the RENBTC-A dust
+        VatAbstract(MCD_VAT).file(RENBTC_ILK, "dust", 100 * RAD);
```

Comment:
@godsflaw are we sticking with dust of 100 on Kovan or should this be 500 now?

Reply:
I think 100 is fine for `kovan`.